deconnable_everything.md

```diff
@@ -0,0 +1,107 @@
+  Exotic machinery found in atmos or the handful of generators should gain exotic materials logically.
+  - ex: Big Scrubbers yielding plasteel after some time of deconstructing
+ 
+ Deconning objects should ALWAYS give more rewards than simply destroying
+
+3. Give some of the more used objects actual recipes to build them back again
+  - If you are able to deconstruct it, you should be able to build it again
+  - Things like hydroponics trays that are commonly used so entire roles wont get screwed because it got deconstructed.
+  - Not everything should have it, of course. There's already enough snowflake objects, we dont need more.
+
+4. Give crafting recipes to stuff that SHOULD have it but do not
+  - Things like wooden tables, pool tables, carpets,  etc. Why on earth isnt there an easy way to make them. MT buff.
+<!-- This is a numbered list clearly detailing your goals for the feature. As per usual, this should be a mixture of both why and what. -->
+
+## Non-goals
+
+1. Nothing should be affecting balance in an adverse way
+  - We obviously don't want people getting entire stacks of plasteel from 1 area.
+  - This is not a major balance affecting MR and shouldn't be.
+
+2. Intentionally indestructable objects should not be touched
+  - this has been repeated throughout this doc but putting it here so someone won't say *"SUPER MARINE/XENO BUFF THEY CAN DESTROY HULL WALLS!!!111"*
+<!-- Just like goals, but the opposite! Every feature has boundaries it won't step over. These should be written as if they start with "We will not...". -->
+
+## Content
+
+1. Rework nearly all unintentional indestructable things into being destructable and deconstructable
+
+  Quite self explanitory, every month a new cheese obj comes out that has to be banned from APC/Dropship or made unmovable/slashable for xenos. Putting our foot down
+  and focusing on ending it will greatly improve QoL with only minimal impact on balance and a whole lot of sanity saved in making bandaid fixes.
+  
+  ***EVERYTHING that is unintentionally indesdtructable NEEDS to be destructable***
+  
+  Computers & vendors will be an issue since some of them SHOULD not be deconnable/destructable like intel computers, but thats something that could easily be fixed with a subtype/var
+
+  Things like crates which contain rewards inside should be destructable with bullets/melee, but have a 50% chance of destroying contents
+
+  This could be done by giving everything health vars and damage procs like seen in showcases or barricades. 
+  
+  Some things that were given snowflake anchoring/indestructable could be taken a look at again and be made deconstructable again.
+  
+  There are potential balance ramifications coming from this due to being more freedom of movement in the battlefield, but I believe freedom of movement is good.
+  *A 15 foot rav shouldn't die because a dinky little filing cabinet blocked their way*
+
+2. Give destructable objects deconstruction paths.
```

Comment:
Implementation detail. Don't include code-like words in design documents, a designer does not care about deconstruction paths. Reword this into 'Give destructible objects a thing they break down into'